src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerWorkspace.cs

```diff
@@ -67,7 +67,14 @@ ValueTask ILspWorkspace.UpdateTextIfPresentAsync(DocumentId documentId, SourceTe
         return this.ProjectSystemProjectFactory.ApplyChangeToWorkspaceAsync(
```

Comment:
don't personally love this.  maybe thsi have been UpdateDocumentText and UpdateAdditionalDocumentText.  But i can live with this for now.